transaction-ballerina/Dependencies.toml

```diff
@@ -1,34 +1,34 @@
 [[dependency]]
```

Comment:
Need to do this(https://github.com/ballerina-platform/module-ballerina-xslt/commit/e7f4467a6d1d886af838cee28786bfdee63960d4) kind of change to revert back the `Dependency.toml` to the original state like `Ballerina.toml`.